index.js

```diff
@@ -61,6 +62,13 @@ const init = async () => {
     delete require.cache[require.resolve(`./events/${file}`)];
   });
 
+  const backgroundFiles = await readdir("./background/");
+  client.log("log", `Loading a total of ${evtFiles.length} background workers.`);
```

Comment:
Please change this to ``client.logger.log(`Loading a total of ${evtFiles.length} background workers.`);``